Package.swift

```diff
@@ -19,6 +19,27 @@ let package = Package(
             name: "AppStoreConnect-Swift-SDK",
             dependencies: ["URLQueryEncoder"],
             path: "Sources",
-            exclude: ["OpenAPI/app_store_connect_api_2.0_openapi.json"])
+            exclude: ["OpenAPI/app_store_connect_api_2.0_openapi.json"]
+        ),
+        .binaryTarget(
+            name: "create-api",
+            url: "https://github.com/CreateAPI/CreateAPI/releases/download/0.0.5/create-api.artifactbundle.zip",
```

Comment:
Does this mean we have to manually update this to the latest release by changing this URL?

Reply:
Yep, I haven't quite worked out what the optimal flow is just yet but other tools (SwiftGen) take this approach where you just define a target and keep it updated yourself. 

I'm adding a bit more documentation here if it helps: https://github.com/CreateAPI/CreateAPI/blob/ln/documentation/Docs/SwiftPackagePlugins.md

In the future, it might be that we expose this as a Package in its own repo (i.e https://github.com/CreateAPI/create-api-cli.git) so that you can add it like any other dependency, but TBH I haven't explored enough to know if that's doable, or the right approach 😄